downstream/modules/platform/proc-change-llm-model-chatbot.adoc

```diff
@@ -0,0 +1,44 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="proc-change-llm-model_{context}"]
+
+= Changing your LLM model 
+
+[role="_abstract"]
+
+If you have already deployed {AAPchatbot} but want to change your LLM model, you can use one of the following methods:
+
+* Create a new chatbot configuration secret for the new LLM model. 
```

Comment:
@sayjadha The new short descriptions for AEM won't allow lists, so you could either make this a paragraph now or later as a separate CQA PR.

Reply:
Made the edit, thank you.